releasenotes/notes/provider-methods-6ca5f5fa09e270f4.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    Methods ``jobs()``, ``job``(), ``job_ids()``, and ``my_reservations()`` have been added to
```

Comment:
Can we make all these methods as links? example: :meth:`~qiskit_ibm_provider.IBMProvider.jobs`